devenv/docker/blocks/openldap-mac/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# Fork of https://github.com/dinkel/docker-openldap
+
+FROM osixia/openldap
+
+LABEL maintainer="Grafana team <hello@grafana.com>"
+
+ENV OPENLDAP_VERSION 2.4.57
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+        slapd=${OPENLDAP_VERSION}* \
+        ldap-utils && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
This is not included in `osixia/openldap`? Why do we need to update/install instead of relying on the docker image updates

Context: haven't checked `osixia/openldap`'s content

Reply:
You are right @Jguer !!! Those 2 libraries are already included at `osixia/openldap`. I'll change this.

Reply:
I have removed the libraries as requested since they are already installed by the `osixia/openldap` container.